### PR TITLE
Add test code for Delete and Check

### PR DESCRIPTION
**Pull request by FeliciaBeatrice**

### CODE REVIEW

File: src/main/java/seedu/duke/command/check/CommandCheckClient.java [hunk 1]
```diff
@@ -17,9 +17,9 @@ public class CommandCheckClient extends CommandCheck {
     private int clientIndex;
 
     /**
-     * Constructs an instance of CommandCheckProperty.
+     * Constructs an instance of CommandCheckClient.
      *
-     * @param commandCheckClientDetails Parsed property index from the user's input.
+     * @param commandCheckClientDetails Parsed client index from the user's input.
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/command/check/CommandCheckClient.java:22]: Good job following the convention.

File: src/test/java/seedu/duke/CheckTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    public static final String COMMAND_DETAILS_EXTRA_FLAG = "n/1 i/1";
+
+
+    @Test
+    void checkForMissingFlags_missingCheckClientFlag_exceptionThrown() {
+        ClientList clientList = new ClientList();
+        CommandCheckClientParser commandCheckClientParser =
+                new CommandCheckClientParser(COMMAND_DETAILS_MISSING_FLAG, clientList);
+        assertThrows(CheckClientMissingFlagException.class, commandCheckClientParser::parseCommand);
+    }
```
Comment (wilsonngja) [inline src/test/java/seedu/duke/CheckTest.java:24]: Good that you covered the corner case

**Review by wilsonngja — COMMENTED** [review]
Good implementation of test code. LGTM

File: src/test/java/seedu/duke/CheckTest.java (hunk 2 above)
Comment (OVReader) [reply src/test/java/seedu/duke/CheckTest.java:24]: Agreed.

**Review by OVReader — APPROVED** [review]
Good job on adding test code for delete! LGTM.
